pkg/contracts/constants.go

```diff
@@ -54,11 +54,13 @@ func (t HashType) Validate() bool {
 type KeyAlgorithm string
 
 const (
-	KeyEd25519 KeyAlgorithm = "ed25519"
+	KeyEd25519        KeyAlgorithm = "ed25519"
+	KeyEcdsaX509      KeyAlgorithm = "ecdsa-x509"
+	KeyEcdsaSepc256k1 KeyAlgorithm = "ecdsa-sepc256k1"
```

Comment:
shouldn't this be `ecdsa-secp256k1` instead of `ecdsa-sepc256k1`? I believe this typo is propagated throughout this PR.

Reply:
Thank you for this. I've gone through and corrected several typos within the PR